dockerfiles/Dockerfile.router

```diff
@@ -1,39 +1,43 @@
 ARG DEBUG_IMAGE
 # Build is required to extract the release files
-FROM --platform=linux/amd64 alpine:latest AS build
+FROM --platform=linux/amd64 debian:bullseye-slim AS build
 
-ARG ROUTER_RELEASE
+ARG ROUTER_RELEASE=latest
 
-# Pull release from GH
-ADD https://github.com/apollographql/router/releases/download/v${ROUTER_RELEASE}/router-${ROUTER_RELEASE}-x86_64-unknown-linux-gnu.tar.gz /tmp/router.tar.gz
+WORKDIR /dist
 
-WORKDIR /tmp
+# Install curl
+RUN \
+  apt-get update -y \
+  && apt-get install -y \
+    curl \
+  && rm -rf /var/lib/apt/lists/*
 
-# router.tar.gz extracts to "dist"
-RUN tar xvzf router.tar.gz -C /
+# Run the Router downloader which puts Router into current working directory
+RUN curl -sSL https://router.apollo.dev/download/nix/${ROUTER_RELEASE}/ | sh
```

Comment:
I'm assuming no certificate issues with debian:bullseye-slim. i.e.: no need to install ca-certificates.

Reply:
I think `curl` just actually brings in `ca-certificates` now.